Ports/nesalizer/package.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=nesalizer
-version=git
-workdir=${port}-master
-makeopts=("CONF=release" "EXTRA=-I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2")
-files="https://github.com/SerenityPorts/nesalizer/archive/master.zip nesalizer-master.zip"
+version=5bb045845a5cc829a78b7384f848fdd886cd98c8
+archive_hash=4282cb0e4af0585af4a594dfa30b2e350dd0efc39e5bc2d8312f637f50397107
```

Comment:
Please just inline this like everywhere else :)

Reply:
Sure, I'll do this from now on. I also squashed the patches into just two patches.